asreviewcontrib/datatools/dedup.py

```diff
@@ -102,4 +103,59 @@ def drop_duplicates_by_similarity(
     if verbose:
         _print_similar_list(similar_list, data)
 
-    asdata.df = asdata.df[~duplicated].reset_index(drop=True)
\ No newline at end of file
+    asdata.df = asdata.df[~pd.Series(duplicated)].reset_index(drop=True)
+
+
+def deduplicate_data(asdata: ASReviewData, args: Namespace) -> None:
+    initial_length = len(asdata.df)
+
+    if args.pid not in asdata.df.columns:
+        print(
+            f"Not using {args.pid} for deduplication "
+            "because there is no such data."
+        )
+
+    if not args.similar:
+        if args.verbose:
+            before_dedup = asdata.df.copy()
+
+            # retrieve deduplicated ASReview data object
+            asdata.drop_duplicates(pid=args.pid, inplace=True, reset_index=False)
+            duplicate_entries = before_dedup[~before_dedup.index.isin(asdata.df.index)]
+
+            if len(duplicate_entries) > 0:
+                print("Duplicate entries:")
+                for i, row in duplicate_entries.iterrows():
+                    print(f"\tLine {i} - {row['title']}")
```

Comment:
Maybe print the PID as well here? It is also deduplicating based on pid after all.

Reply:
Sure, I will add it to it.